calyx-py/calyx/builder_util.py

```diff
@@ -0,0 +1,190 @@
+# pylint: disable=import-error
+import calyx.builder as cb
+
+
+def insert_adder(
+    comp: cb.ComponentBuilder,
+    cell,
```

Comment:
`name_hint` or something might be better? This confused me for a second into thinking that user passes the adder.